packages/web-twig/src/Resources/components/Breadcrumbs/Breadcrumbs.twig

```diff
@@ -30,8 +30,9 @@
                         href="{{ item.url }}"
                         color="{{ loop.last ? 'secondary' : 'primary' }}"
                         isUnderlined="{{ loop.last != true }}"
-                        aria-current="{{ loop.last ? 'page' : 'false' }}"
-                    >{{ item.title }}</Link>
+                        aria-current="{{ loop.last ? 'page' : 'false' }}">
```

Comment:
i don't like this. Can we freeze twigx version or fix the problem there and avoid this? This way we would spread weird formatting just because of this error and then we would have to format it nicely everywhere when it's fixed.

Reply:
Fixed.